Exiled.API/Features/Player.cs

```diff
@@ -1431,15 +1431,27 @@ public bool RemoveItem(Item item, bool destroy = true)
         /// Hurts the player.
         /// </summary>
         /// <param name="damageHandlerBase">The <see cref="DamageHandlerBase"/> used to deal damage.</param>
-        public void Hurt(DamageHandlerBase damageHandlerBase) => ReferenceHub.playerStats.DealDamage(damageHandlerBase);
+        public void Hurt(DamageHandlerBase damageHandlerBase)
```

Comment:
This method shouldn't contain the cassie message.
It would make anything that uses this method and kills the player to make cassie say that.

Also: If for example you used the other hurt method you changed, and make cassie say something, it then will call this method and make cassie say the same. Therefore you will have repeated the same announcement.

Reply:
ups, my fault